test/src/test_lu_mult.cpp

```diff
@@ -0,0 +1,66 @@
+/// @file test_lu_mult.cpp
+/// @brief Test LU multiplication
+//
+// Copyright (c) 2022, University of Colorado Denver. All rights reserved.
+//
+// This file is part of <T>LAPACK.
+// <T>LAPACK is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include <catch2/catch.hpp>
+#include <plugins/tlapack_stdvector.hpp>
+#include <tlapack.hpp>
```

Comment:
you should also include `<plugins/tlapack_legacyArray.hpp>`

Your code currently works because `tlapack.hpp` also happens to include, but you may get in trouble if that changes in the future.

Reply:
Thanks!